libraries/shared/executor/executor.go

```diff
@@ -0,0 +1,104 @@
+// VulcanizeDB
+// Copyright © 2019 Vulcanize
+
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package executor
+
+import (
+	"fmt"
+	"github.com/vulcanize/vulcanizedb/libraries/shared/transformer"
+	"github.com/vulcanize/vulcanizedb/libraries/shared/watcher"
+	"plugin"
+	syn "sync"
```

Comment:
I don't know why we use this alias but I used it to be consistent...

Reply:
I think the reason for the alias is usually if we have another import of the same name or a pressing need to use the same word (e.g. `sync`) as a variable. Generally a fan of removing these aliases when we can